cardano-node/src/Cardano/Tracing/Config.hs

```diff
@@ -138,6 +143,11 @@ data TraceSelection
   , traceTxOutbound :: OnOff TraceTxOutbound
   , traceTxSubmissionProtocol :: OnOff TraceTxSubmissionProtocol
   , traceTxSubmission2Protocol :: OnOff TraceTxSubmission2Protocol
+  -- TODO: Non P2P
```

Comment:
What this todo stands for?

Reply:
These TODOs exist in order for us to remember what was added in `p2p-master` that refers to `master` and then remove when changing to just `P2P`